molecule/default/molecule.yml

```diff
@@ -0,0 +1,44 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint: |
+  yamllint .
+  ansible-lint .
+platforms:
+  - name: blackbox_exporter_debian10
+    groups:
+      - blackbox_exporter_group
+    image: ${MOLECULE_DISTRO:-debian:buster-slim}
+    privileged: false
+    capabilities:
+      - SYS_ADMIN
+    tmpfs:
+      - /tmp
+      - /run
+      - /run/lock
+    volumes:
+      - '/sys/fs/cgroup:/sys/fs/cgroup:ro'
+    command: '/lib/systemd/systemd'
+    stop_signal: 'RTMIN+3'
+
+  - name: blackbox_exporter_debian9
```

Comment:
Shouldn't Debian 9 testing be achieved via MOLECULE_DISTRO var?

Reply:
done :)